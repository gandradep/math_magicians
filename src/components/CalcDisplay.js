import React from 'react';
import PropTypes from 'prop-types';

class CalcDisplay extends React.PureComponent {
  render() {
    const { classy, name } = this.props;
    return <span className={classy}>{name}</span>;
  }
}
CalcDisplay.propTypes = {
  name: PropTypes.string.isRequired,
  classy: PropTypes.string.isRequired,
};

export default CalcDisplay;
