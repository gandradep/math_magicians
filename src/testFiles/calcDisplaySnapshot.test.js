import renderer from 'react-test-renderer';
import CalcDisplay from '../components/CalcDisplay';

it('CalcDisplay renders correctly', () => {
  const tree = renderer
    .create(<CalcDisplay />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
