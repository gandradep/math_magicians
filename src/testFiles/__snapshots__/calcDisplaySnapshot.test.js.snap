// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalcDisplay renders correctly 1`] = `<span />`;
